src/main/java/util/DBConnector.java

```diff
@@ -0,0 +1,32 @@
+package util;
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DBConnector {
+
+    private static final String dbUrl = "jdbc:mysql://localhost:3306/db" +
```

Comment:
```suggestion
    private static final String DB_URL = "jdbc:mysql://localhost:3306/db" +
```
the same with login and password